.github/workflows/acceptance-tests.yml

```diff
@@ -115,3 +115,4 @@ jobs:
       mongodb_atlas_enable_preview: ${{ vars.MONGODB_ATLAS_ENABLE_PREVIEW }}
       azure_private_endpoint_region: ${{ vars.AZURE_PRIVATE_ENDPOINT_REGION }}
       mongodb_atlas_rp_org_id: ${{ inputs.atlas_cloud_env == 'qa' && vars.MONGODB_ATLAS_RP_ORG_ID_QA || vars.MONGODB_ATLAS_RP_ORG_ID_DEV }}
+      mongodb_atlas_flex_project_id: ${{ vars.MONGODB_ATLAS_FLEX_PROJECT_ID }}
```

Comment:
@cveticm @oarbusi we should consider that these tests will fail in QA. 
Do we know if flex cluster functionality is available in QA? If so we can create a project in our QA org and configure that as well.

Reply:
Flex cluster feature is not available in QA and this tests won't be running in QA because it's all in a feature branch. We've created a [ticket](https://jira.mongodb.org/browse/CLOUDP-280223) to ensure tests run correctly in QA when the feature is available in QA